CodeReviews/CodeReviewBadPractices.md

```diff
@@ -1,18 +1,20 @@
+When reviewing others code it is best to remember that it is not a competition to see who can code better. Software developers are often proud of their solutions and take criticism personally on occasion. Reviews help us see how others thought process occurs and feedback given should be constructive uplifting and insightful to help others program more efficiently. Be careful to always provide feedback that does not question the person but questions how and why they created a solution the way they did. However if the developer has done something right and you found how they've written their code or how they implemented a functionality to be good/clever let them know.
 
 <p align="center">
 <img src="https://miro.medium.com/max/1400/1*D9YKlFGZKiYoR57tUTpCrg.jpeg" alt="T-Shirt Size Image" width="400">
 <p>
 
 * **Styling Conventions**  
-The team should decide on how they want to style their code, your teams code should be consistent and look relatively the same. Not having a style guide placed can lead to changes being disapproved during code reviews due to nitpicking about the slightest of cosmetic issues. As someone submitting your code for review you should make sure that it already conforms to the standards you as a team agreed upon. Style-guide violations will be caught in reviews but are a waste of time for both you and the reviewers involved.
-
+The team should decide on how they want to style their code, your teams code should be consistent and look relatively the same. Not having a style guide placed can lead to changes being disapproved during code reviews due to nitpicking about the slightest of cosmetic issues. Style-guide violations will be caught in reviews but are a waste of time for both you and the reviewers involved.
 
 * **Meaningful comments**  
-If a person makes an error while programming it is almost certain that this error has been repeated across in other parts. When reviewing others code do not point out every occurrence of the error instead leave a detailed not explaining the overall issue. It is essential to reply to all comments in order to ensure that the review process is actually being completed.
+When reviewing others code do not point out every occurrence of an error instead leave a detailed note explaining the overall issue. 
+Ensure there is a balance between simply pointing out problems for them to fix and giving explicit instructions on how to fix. As giving explicit instructions can deny them the opportunity to learn from their mistakes.
+It is essential to reply to all comments in order to ensure that the review process is actually being completed. 
+
 
 <p align="center">
 <img src="https://miro.medium.com/max/1400/1*UbeT74oIf3JlX1YRu90EKA.jpeg" alt="T-Shirt Size Image" width="400" center>
```

Comment:
Should reference the images in the repo and modify alternative text to a more appropriate name